compiler/ballerina-lang/src/main/resources/interop-dependencies.properties

```diff
@@ -0,0 +1,7 @@
+ballerina/http\:2.10.7=Caller,Remote,Local,Link,MutualSslHandshake,HeaderValue,Request,Response,Entity,RequestCacheControl,Headers,RequestContext,JwtDecoder,ErrorDetail,ResponseCacheControl,HttpFuture,PushPromise,ApplicationResponseError,DefaultErrorDetail,Error,ListenerError,ClientError,HeaderNotFoundError,PayloadBindingError,InboundRequestError,OutboundResponseError,GenericListenerError,InterceptorReturnError,InternalInterceptorReturnError,HeaderBindingError,InternalHeaderBindingError,HeaderValidationError,InternalHeaderValidationError,NoContentError,PayloadBindingClientError,InternalPayloadBindingListenerError,PayloadValidationError,PayloadValidationClientError,InternalPayloadValidationListenerError,QueryParameterBindingError,InternalQueryParameterBindingError,QueryParameterValidationError,InternalQueryParameterValidationError,PathParameterBindingError,InternalPathParameterBindingError,RequestDispatchingError,InternalRequestDispatchingError,ServiceDispatchingError,ResourceDispatchingError,ListenerAuthError,ListenerAuthnError,ListenerAuthzError,InternalListenerAuthnError,InternalListenerAuthzError,OutboundRequestError,InboundResponseError,ClientAuthError,ResiliencyError,GenericClientError,Http2ClientError,SslError,ApplicationResponseError,UnsupportedActionError,MaximumWaitTimeExceededError,CookieHandlingError,ClientConnectorError,ClientRequestError,RemoteServerError,FailoverAllEndpointsFailedError,FailoverActionFailedError,UpstreamServiceUnavailableError,AllLoadBalanceEndpointsFailedError,CircuitBreakerConfigError,AllRetryAttemptsFailed,IdleTimeoutError,LoadBalanceActionError,InitializingOutboundRequestError,WritingOutboundRequestHeadersError,WritingOutboundRequestBodyError,InitializingInboundResponseError,ReadingInboundResponseHeadersError,ReadingInboundResponseBodyError,InitializingInboundRequestError,ReadingInboundRequestHeadersError,ReadingInboundRequestBodyError,InitializingOutboundResponseError,WritingOutboundResponseHeadersError,WritingOutboundResponseBodyError,Initiating100ContinueResponseError,Writing100ContinueResponseError,InvalidCookieError,ServiceNotFoundError,InternalServiceNotFoundError,BadMatrixParamError,InternalBadMatrixParamError,ResourceNotFoundError,InternalResourceNotFoundError,ResourcePathValidationError,InternalResourcePathValidationError,ResourceMethodNotAllowedError,InternalResourceMethodNotAllowedError,UnsupportedRequestMediaTypeError,InternalUnsupportedRequestMediaTypeError,RequestNotAcceptableError,InternalRequestNotAcceptableError,ResourceDispatchingServerError,InternalResourceDispatchingServerError
+ballerina/sql\:1.11.1=BatchExecuteErrorDetail,DatabaseErrorDetail,Error,DatabaseError,BatchExecuteError,NoRowsError,ApplicationError,DataError,TypeMismatchError,ConversionError,FieldMismatchError,UnsupportedTypeError
```

Comment:
No new line at end of file ([ref](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log))

Reply:
Fixed in 75787839cdedf63d7e0fb3f54e5ba1e06282532c.